install.sh

```diff
@@ -1388,19 +933,24 @@ IM_PROBES_FILES="\
 
 IM_PROBES_LIB_FILES="\
     src/im_mad/remotes/lib/kvm.rb \
-    src/im_mad/remotes/lib/lxd.rb \
     src/im_mad/remotes/lib/lxc.rb \
     src/im_mad/remotes/lib/linux.rb \
-    src/im_mad/remotes/lib/firecracker.rb\
     src/im_mad/remotes/lib/numa_common.rb \
     src/im_mad/remotes/lib/probe_db.rb \
-    src/im_mad/remotes/lib/vcenter_monitor.rb \
-    src/im_mad/remotes/lib/vcenter_cluster.rb \
-    src/im_mad/remotes/lib/vcenter_monitor_vms.rb \
     src/im_mad/remotes/lib/monitord_client.rb \
     src/im_mad/remotes/lib/domain.rb \
     src/im_mad/remotes/lib/process_list.rb"
 
+IM_PROBES_LIB_PYTHON_FILES="\
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

IM_PROBES_LIB_PYTHON_FILES appears unused. Verify use (or export if used externally).